expo/android/src/main/java/land/gno/gnonative/GnonativeModule.kt

```diff
@@ -35,6 +35,17 @@ class GnonativeModule : Module() {
       rootDir = context!!.filesDir
     }
 
+    OnDestroy {
+      try {
+        if (bridgeGnoNative != null) {
+          bridgeGnoNative?.close()
```

Comment:
Just a question, are you sure the `?` is mandatory in `bridgeGnoNative?.close()`? Because you have already tested the nullity before.

Reply:
Yes, it's mandatory, I tried without the ? and it gave an error about "nullable type".